deployments/k8s/templates/deployment.yaml

```
@@ -40,6 +40,19 @@ spec:
             - name: http
               containerPort: {{ .Values.ports.http }}
               protocol: TCP
+          livenessProbe:
+            httpGet:
+              path: /
+              port: 80
```

I'm not very familiar with k8s configuration, so I'm leaving a more detailed review to @smoya and @magicmatatjahu  🙂
My only comment is if we should use the port that we define above, instead of hardcoding port `80`, on both `readinessProbe` and `livenessProbe`:
```suggestion
              port: http
```

---

Yeah, @BOLT04 You are right. @smoya & @magicmatatjahu  should I change it?

---

Yes, you are right. Better to use the name of the port, which in this case is called `http` 👍  nice catch @BOLT04